unit-tests/dds/test-librs-device-properties.py

```diff
@@ -44,7 +44,7 @@
             test.check_equal( len(sensor.get_stream_profiles()), 104 ) # As measured running rs-sensor-control example
         if test.check( 'RGB Camera' in sensors ):
             sensor = sensors['RGB Camera']
-            test.check_equal( len(sensor.get_stream_profiles()), 225 ) # As measured running rs-sensor-control example
+            test.check_equal( len(sensor.get_stream_profiles()), 64 ) # As measured running rs-sensor-control example
```

Comment:
I remember when Flexible was shorter in max length, and we though the number of profiles would be high. I guess it could still happen, but would now likely be shorter...
:)

Reply:
But still with `options` and `intrinsics` it can be about 4K

Reply:
Hmm true.